index.ts

```diff
@@ -30,6 +30,7 @@ export { AuthHelperService } from './src/app/launcher/service/auth-provider.serv
 export { HelperService } from './src/app/launcher/service/helper.service';
 export { TokenProvider } from './src/app/service/token-provider';
 export { Config } from './src/app/service/config.component';
+export { StaticInjector } from './src/app/launcher/shared/telemetry.decorator';
```

Comment:
it doesn't need to be exported right

Reply:
@edewit i need to get the instance from the fabric8-ui right ?